Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM python:3.8
 
-WORKDIR /
+ARG UID
+ARG GID
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
 
+WORKDIR /
 # install requirements
 COPY ./requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
 # create a new user and use it.
-RUN useradd -M -u 1001 nonrootuser
-USER nonrootuser
+RUN mkdir -p /home/iceesuser
+RUN groupadd --system -g $GID iceesuser && useradd --system -g iceesuser --shell /bin/bash -u $UID --home /home/iceesuser iceesuser
 
+WORKDIR /home/iceesuser
 # set up API things
 COPY ./icees_api icees_api
 COPY ./main.sh main.sh
 COPY ./examples examples
 
+RUN chown -R iceesuser:iceesuser /home/iceesuser
+
+USER iceesuser
+WORKDIR /home/iceesuser
```

Comment:
I believe the working directory is already here from line 19?

Reply:
You are right. I removed the duplicate WORKDIR statement on line 28 and tested it on ebcr0 and everything worked, so I removed it from my latest commit.